LambdaEngine/Source/Resources/ResourceManager.cpp

```diff
@@ -406,6 +419,213 @@ namespace LambdaEngine
 		return guid;
 	}
 
+	void ResourceManager::CombineMaterials(Material* pMaterial, GUID_Lambda guid)
+	{
+		// Find largest texture size
+		uint32 largestHeight;
+		uint32 largestWidth;
+		largestWidth	= std::max(pMaterial->pMetallicMap->GetDesc().Width, std::max(pMaterial->pRoughnessMap->GetDesc().Width, pMaterial->pAmbientOcclusionMap->GetDesc().Width));
+		largestHeight	= std::max(pMaterial->pMetallicMap->GetDesc().Height, std::max(pMaterial->pRoughnessMap->GetDesc().Height, pMaterial->pAmbientOcclusionMap->GetDesc().Height));
+
```

Comment:
Consider assigning largestHeight and largestWidth directly instead. I think it is clear enough.

Reply:
fixed